backend/json.go

```diff
@@ -0,0 +1,82 @@
+package backend
+
+import (
+	"unsafe"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	jsoniter "github.com/json-iterator/go"
+)
+
+// This will make sure jsoniter uses a fast JSON serialization strategy
+var (
+	_ = initEncoders()
```

Comment:
@FZambia @aknuds1 any thoughts on this approach?  

I suppose we should use the same encoder for the standard Marshal functions?  (currently do not exist)

Reply:
This seems a bit scary to me since I don't understand what's going on under the hood. If sth will go wrong it will take a lot of time to fix (or we need to invest more time into understanding how it works). I think we can use jsoniter without registering types (maybe loosing some performance)?

Reply:
Another thought - this leads to the situation where `jsoniter` goes out of SDK package since SDK user should import `jsoniter` to serialize type to JSON. Is this correct? I think `jsoniter` should not be exposed in any form and better stay implementation details.